### DISCUSSION TITLE
Please reply and solve these problems🙏🙏🙏🙏

### DISCUSSION THREAD

**Discussion by Fatma1412** [community/community#62429] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Every time I post a Twitter tweet, it appears to me that it is not possible to view the tweets of this protected user. Please, what is the solution to this problem?

**Comment by DSDmark** [comment]
Hi @Fatma1412,

This discussion is related to **twitter community**. So, it's better if you do that post there. This community is related to **GitHub** and its products and git related repository.

**Comment by Fatmaana** [comment]
is there a solution ?  I need to apply
